services/web/server/src/simcore_service_webserver/catalog_api_models.py

```diff
@@ -65,49 +30,115 @@ def get_formatted_unit(data: dict):
     return [None, None]
 
 
-class _CommonApiExtension(BaseModel):
+def json_dumps(v, *, default=None) -> str:
+    # orjson.dumps returns bytes, to match standard json.dumps we need to decode
+    return orjson.dumps(v, default=default).decode()
```

Comment:
Any special reason for using orjson instead of json?

Reply:
> Any special reason for using orjson instead of json?

That was discussed several PRs ago. SEE thread [here](https://github.com/ITISFoundation/osparc-simcore/pull/2145#discussion_r576339850). 

Basically orjson is faster and has more pre-defined encoders.